shell_scripts/lukso

```diff
@@ -621,14 +621,20 @@ start_validator() {
   if [[ -n $TEMPPASSFILE ]]; then
     # Wait until validator starts
     TRIES=0
-    while [[ ! $(lsof -t -i:7000) ]]; do
+    while [[ $TRIES -lt 5 ]]; do
```

Comment:
I think you can prepare `const` variable at the begining of this script or just declare like:
```
##### network info
CHAIN_ID=""
NETWORK_ID=""
#####
```

Reply:
done